### PR TITLE
Middleware and Success Port Connection Listening

### PR DESCRIPTION
**Pull request by GundamMerlin**
Up for review

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const routes = require("./routes");
+
+const app = express();
+const PORT = process.env.PORT || 4567
+//Middleware
+app.use(cors());
+app.use(express.jason());
+app.use(logger("dev"));
+//
+app.use()
```
Comment (foremanlb) [inline server.js]: add "  '/api', routes " to the () of app.use

File: server.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+db.on("error", console.error.bind(console, "MongoDB connection error:"));
```
Comment (foremanlb) [inline server.js]: move above the app.uses please

File: server.js (hunk 1 above)
Comment (GundamMerlin) [reply server.js]: Done

File: server.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+const routes = require("./routes");
+
+const app = express();
+const PORT = process.env.PORT || 4567
+//Middleware
+app.use(cors());
+app.use(express.jason());
+app.use(logger("dev"));
+//
+app.use('/api')
```
Comment (foremanlb) [inline server.js]: , routes after api

File: server.js (hunk 2 above)
Comment (GundamMerlin) [reply server.js]: Done

File: server.js (hunk 3 above)
Comment (GundamMerlin) [reply server.js]: done

File: server.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+db.on("error", console.error.bind(console, "MongoDB connection error:"));
+
+const app = express();
+const PORT = process.env.PORT || 4567
+//Middleware
+app.use(cors());
+app.use(express.jason());
+app.use(logger("dev"));
+//
+app.use('/api/routes')
```
Comment (foremanlb) [inline server.js:16]: app.use('/api', routes)

**Review by foremanlb — COMMENTED** [review]
check comments

File: controllers/tutors.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+const Tutor = require('../models/tutor.js')
+const db require('../db')
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken")
+
+db.on('error', console.error.bind(console, 'MongoDB connection error'))
+
+const SALT_ROUNDS = 11;
+const TOKEN_KEY
```
Comment (wjb108) [inline controllers/tutors.js:9]: put token string in

**Pull request closed without merging**